dbt/adapters/snowflake/impl.py

```diff
@@ -180,7 +180,13 @@ def submit_python_job(self, parsed_model: dict, compiled_code: str):
         database = getattr(parsed_model, "database", self.config.credentials.database)
         identifier = parsed_model["alias"]
         proc_name = f"{database}.{schema}.{identifier}__dbt_sp"
-        packages = ["snowflake-snowpark-python"] + parsed_model["config"].get("packages", [])
+        packages = parsed_model["config"].get("packages", [])
```

Comment:
This make sure that if user also specify `snowflake-snowpark-python` in the config, we will still run everything well